models/group_model.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+import (
+	"github.com/kamva/mgm/v3"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Group struct {
+	mgm.DefaultModel `bson:",inline"`
+	Users            []primitive.ObjectID `bson:"users"`
+	Drift            []primitive.ObjectID `bson:"drift"`
```

Comment:
See comment above. Again, I don't know if we should have a separate collection for Groups especially since people may be drifting with various groups of friends